lib/web_ui/test/canvaskit/semantics_test.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 @TestOn('chrome || safari || firefox')
+library;
```

Comment:
Why are these empty `library;` needed? Is it because of the `@TestOn`?

Reply:
Yeah, previously, the TestOn was technically annotating the next import statement.